domains/emissions/Emissions.API/Tests/Resources/datasync_measurements.json

```diff
@@ -1,16 +1,16 @@
 [
-    {
-        "gsrn": "571313121223234323",
-        "dateFrom": "1609455600",
-        "dateTo": "1609459200",
-        "quantity": 1250,
-        "quality": "measured"
-    },
-    {
-        "gsrn": "571313121223234323",
-        "dateFrom": "1609459200",
-        "dateTo": "1609462800",
-        "quantity": 2520,
-        "quality": "measured"
-    }
-]
+  {
+    "gsrn": "571313121223234323",
+    "dateFrom": "2020-12-31T23:00:00.000Z",
+    "dateTo": "2021-01-01T00:00:00.000Z",
```

Comment:
Have the format from datasync changed? This seems wrong..

Might be because of changes to Models/Measurement.cs that you have changed the test.
But this is a response from DataSync, and that has not changed, so the test shouldn't either.

Reply:
I mistook the Measurement DTO for a class internal to the Emissions Api. 
Should we use longs for dates in DataSync, since it's delivered to our other Apis via json?
If so, I will change this back to using longs.

Reply:
On a related note: Shouldn't the Measurement DTO come from a nuget package published by the DataSync Api? Otherwise, there should be an integration test to ensure that these two apis don't stop working together.